A320-main.xml

```diff
@@ -3903,6 +3903,30 @@
                         <property>/sim/view/config/back-direction-deg</property>
                     </binding>
                 </mod-shift>
+            </key>
```

Comment:
[Formating guidlines](https://github.com/legoboyvdlp/A320-family/blob/dev/CONTRIBUTING.md#formatting-guidelines) are not met.

Reply:
Sorry for this. I have no idea why this happened. Now I have corrected the formatting, but because of that, there are a lot of changes. Is that ok so, or shall I start with the original file from new?